tests/e2e/testRunner.js

```diff
@@ -1,3 +1,5 @@
+import launchApp from './utils/launchApp';
```

Comment:
WHy this change?

Reply:
Because this launch app test is converted into es6 module so I also changed it here

Reply:
This is a nodeJS only app, you can't use import statements here (or we need to transpile the code)

This caused a regression of breaking all e2e tests:

```
(node:3308) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
/tmp/scratcholkuFS.scratch/test-packageo7zoRO/zip/e2e/testRunner.js:1
import launchApp from './utils/launchApp';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1126:15)
    at Module._compile (node:internal/modules/cjs/loader:1162:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:22:47
 ```

Reply:
Its not getting converted to an es6 module

Reply:
ahhh wait, I am just seeing that you changed the scripts in `package.json` to ts-node. Okay, I mean we can work it like this, but this still needs to be addressed on the infra side as well. I will put up a PR